File: Back/src/MedicChat.Domain/model/Paciente.cs
using System;
using System.Collections.Generic;

namespace MedicChat.Domain.model
{
    public class Paciente
    {
        public int Id { get; set; }
        public string Nome { get; set; }
        public string Email { get; set; }
        public int Telemovel { get; set; }
        public string Foto { get; set; }
        public DateTime DataNascimento { get; set; }
        public string Genero { get; set; }
        public string Endereco { get; set; }
        public string CodPostal { get; set; }

        // Propriedade de Navegação!
        public IEnumerable<VideoChat> VideoChats { get; set; }


    }
}

File: Front/MediChat-App/src/app/models/Paciente.ts
import { VideoChat } from './VideoChat';
export interface Paciente {
  id: number;
  nome: string;
  email: string;
  telemovel: string;
  foto: string;
  dataNascimento: Date;
  genero: string;
  endereco: string;
  codPostal: string;
  videoChats: VideoChat;
}
